substrate/state-db/src/lib.rs

```diff
@@ -235,6 +245,20 @@ impl<BlockHash: Hash, Key: Hash> StateDbSync<BlockHash, Key> {
 		}
 	}
 
+	/// Revert all unfinalized blocks with the best block number.
+	/// Returns a database commit or `None` if not possible.
+	/// For archive an empty commit set is returned.
+	pub fn revert_one(&mut self) -> Option<CommitSet<Key>> {
```

Comment:
the method name is `_one`, yet the docs say `all`. This is confusing, which one is it?

Reply:
There may be multiple unfinalized blocks sharing the same number. This reverts a single level of the block tree, including all blocks on that level.

Reply:
Should be read as "revert one block number", but this may include multiple actual blocks.

Reply:
Gotcha. The naming is still a bit confusing, however, I don't have any better idea, so go for it.